lib/Conversion/ImportVerilog/ImportVerilog.cpp

```diff
@@ -66,6 +66,14 @@ Location Context::convertLocation(slang::SourceLocation loc) {
                                         getBufferFilePath, loc);
 }
 
+void Context::pushLValue(mlir::Value *lval) { lvalueStack.push_back(lval); }
+void Context::popLValue() { lvalueStack.pop_back(); }
+mlir::Value *Context::getTopLValue() const {
+  if (lvalueStack.empty())
+    return nullptr;
+  return lvalueStack.back();
+}
+
```

Comment:
You shouldn't need explicit lvalue handling initially -- see #30. We might want to introduce a more rigorous distinction at some point though.